types/hostList.go

```diff
@@ -58,9 +59,21 @@ func (h *HostsList) DecodeMapstructure(value interface{}) error {
 		}
 		*h = list
 	case []interface{}:
-		*h = decodeMapping(v, ":")
+		*h = decodeMappingSeps(v, []string{"=", ":"})
```

Comment:
could change `decodeMapping` to accept a variatic `sep` parameter so we don't need another implementation

Reply:
Ah, thank you - that's a better idea! Will do ...